flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducerITCase.java

```diff
@@ -174,6 +174,7 @@ public void testRestoreToCheckpointAfterExceedingProducersPool() throws Exceptio
 
     /** This test hangs when running it in your IDE. */
     @Test
+    @Ignore
```

Comment:
What is the reason that this test is ignore? @PatrickRen  Do you have a followup to enable it again?